arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -831,6 +831,14 @@
                         <string>Bristol3DModel</string>
                     </array>
                 </dict>
+                <dict>
+                    <key>displayName</key>
+                    <string>Terrain Exaggeration</string>
+                    <key>storyboardName</key>
+                    <string>TerrainExaggeration</string>
+                    <key>descriptionText</key>
+                    <string>This sample demonstrates how to use terrain exaggeration with a scene</string>
```

Comment:
How about "This sample demonstrates how to use terrain exaggeration on a surface in a scene"